Src/Common/StringUtilities.cs

```diff
@@ -47,19 +42,21 @@ public static string EnforceMaxLength(string input, int maxLength)
         /// https://github.com/w3c/distributed-tracing/blob/master/trace_context/HTTP_HEADER_FORMAT.md#trace-id
         /// </summary>
         /// <returns>Random 16 bytes array encoded as hex string</returns>
+        [Obsolete("Use Microsoft.ApplicationInsights.Extensibility.W3C.W3CUtilities.GenerateTraceId instead.", true)]
```

Comment:
Will it break compilation? It can be fine if we didn't have this feature in any stable (I remember we didn't), so migrating between stables won't suddenly stop compiling, right?